ecosystem/daos/dao.py

```diff
@@ -4,7 +4,6 @@
 File structure:
 
     root_path
-    ├── members.json  # compiled file; don't edit manually
     └── members
```

Comment:
In a follow up, would it make sense to flatten this structure?

Reply:
True, I actually think the members data should probably live under `~/members/*.toml` rather than where it is now.